packages/runtime-core/src/component.ts

```diff
@@ -236,6 +239,12 @@ export function validateComponentName(name: string, config: AppConfig) {
   }
 }
 
+export function validateDirectiveName(name: string) {
```

Comment:
This and `isBuiltInDirective` should probably be moved to `directive.ts` instead.